webservices/tasks/__init__.py

```diff
@@ -8,34 +8,34 @@
 
 # Feature and dev are sharing the same RDS box so we only want dev to update
 schedule = {}
-if env.app.get('space_name', 'unknown-space').lower() != 'feature':
-    schedule = {
-        'refresh_materialized_views': {
-            'task': 'webservices.tasks.refresh.refresh_materialized_views',
-            'schedule': crontab(minute=0, hour=9),
-        },
-
-        'reload_all_aos_daily_except_sunday': {
-            'task': 'webservices.tasks.legal_docs.reload_all_aos_when_change',
-            'schedule': crontab(minute=0, hour=1, day_of_week='mon,tue,wed,thu,fri,sat'),
-        },
-
-        'reload_all_aos_every_sunday': {
-            'task': 'webservices.tasks.legal_docs.reload_all_aos',
-            'schedule': crontab(minute=0, hour=1, day_of_week='sun'),
-        },
-
-        'refresh_legal_docs': {
-            'task': 'webservices.tasks.legal_docs.refresh',
-            'schedule': crontab(minute='*/5', hour='10-23'),
-        },
-
-        'backup_elasticsearch_every_sunday': {
-            'task': 'webservices.tasks.legal_docs.create_es_backup',
-            'schedule': crontab(minute=0, hour=4, day_of_week='sun'),
-        },
+# if env.app.get('space_name', 'unknown-space').lower() != 'feature':
+schedule = {
+    # 'refresh_materialized_views': {
```

Comment:
I think this needs to be uncommented before merge.

Reply:
thats correct, will revert this change!